applications/seo.md

```diff
@@ -4,13 +4,13 @@ description: Machine translation for SEO
 
 ### SEO
 
-Machine translation for for **search engine optimization** (SEO) is the translation of website content into the languages in which users are searching.
+Machine translation for **search engine optimization** (SEO) is the translation of [commerce and marketplaces](/applications/commerce-and-marketplaces) websites content into the languages in which users are searching.
```

Comment:
"marketplaces websites content" => "marketplace website content"

Reply:
I guess **SEO** should also be bold

Reply:
In user-generated-content.md, we didn't format the UGC acronym in bold. I understand that acronym and SEO have different relevance, but should we unify their format?

Reply:
Yes, let's unify their format.  Basically, the first mention of the article AND the first mention of synonyms of it should be in bold.  An abbreviation is a synonym of it.